YML-Template.yml

```diff
@@ -24,16 +25,82 @@ Detection:
 - type: IOC
   value: Detection method description
 KnownVulnerableSamples:
-- Filename: FILENAME.sys
-  MD5: ''
-  SHA1: ''
-  SHA256: Your SHA256 hash
-  Signature: ''
-  Date: Time and Date of the file
-  Publisher: File publisher
-  Company: ''
+- Filename: ADV64DRV.sys
```

Comment:
I think the template files should have these fields empty.

Reply:
updated the YAML template now to have example comments but generic values.